proposals/0005-MicroProfile-1.1.md

```diff
@@ -0,0 +1,61 @@
+# MicroProfile 1.1
+
+* Proposal: [MP-0005](0005-MicroProfile-1.1.md)
```

Comment:
I'm not sure, but I think it could be useful as the specs we've got evolve. I think the main problem with the number clash is that proposals aren't being merged quickly enough even after there's broad acceptance, e.g. JWT still isn't merged despite it being widely accepted as "going to happen in some form or another". Most of the discussion on the PR is specific to the API design, which seems like it would fit better in its own repo.